data-source/jsonnet/common/blocks/individual/personal-details/establishment_position.jsonnet

```diff
@@ -23,7 +23,7 @@ local question(title) = {
         {
           label: 'Resident',
           value: 'Resident',
-          description: 'For example, student, member of Armed Forces, patient, detainee',
+          description: 'For example, student, member of armed forces, patient, detainee',
```

Comment:
This sounds odd to me- suggest "member of _the_ armed forces"?

Reply:
I agree that it does sound a little odd - however not sure what the process is for making changes to phrasing like this? Does this change need to be suggested elsewhere?